README.md

```diff
@@ -434,6 +435,7 @@ Running `lerna` without arguments will show all commands/options.
 - `bootstrapConfig.ignore`: an glob that won't be bootstrapped when running the `lerna bootstrap` command.
 - `bootstrapConfig.scope`: an glob that restricts which packages will be bootstrapped when running the `lerna bootstrap` command.
 - `packages`: Array of globs to use as package locations.
+- `registry`: The url of the registry to publish non-private packages to, and to read package metadata from.
```

Comment:
Why not just use a `.npmrc` file in the root of the repository with that config? We'd still need to check it (and pass `--registry <url>` if present) when running npm operations in packages, but that's no different than currently.

Using the `.npmrc` file means we can use `npm config get registry` from the root and get the correct value, simplifying the various commands that need this. It also keeps the `lerna.json` focused, instead of duplicating information already available elsewhere.